rust/monovertex/Cargo.toml

```
@@ -34,10 +34,11 @@ backoff = { path = "../backoff" }
 parking_lot = "0.12.3"
 prometheus-client = "0.22.3"
 kube = "0.94.0"
+log = "0.4.22"
 
 [dev-dependencies]
 tempfile = "3.11.0"
-numaflow = { git = "https://github.com/numaproj/numaflow-rs.git", branch = "main" }
+numaflow = { git = "https://github.com/numaproj/numaflow-rs.git", branch = "handshake" }
```

```suggestion
numaflow = { git = "https://github.com/numaproj/numaflow-rs.git", branch = "main" }
```

eventually we should point to a stable release in crates.io